### ISSUE TITLE
Remove 'skufamily; parameter from azuredeploy.json

### ISSUE THREAD

**Issue by rgardler** [Azure/Moodle#54]
It's late so I don't want to do this now incase I'm missing something, but it looks to me like we can remove some parameters from `azuredeploy.json`. @hosungsmsft  please review the below:

`skufamily` appears only in documentation and has meaningless allowed values

```
$ grep -rnw '.' --include '*.json' -e 'skuFamily'
./azuredeploy.json:257:        "skuFamily": {
./azuredeploy.json:729:            "skuFamily": "[parameters('skuFamily')]",
./nested/mssql.json:50:        "documentation6": " skuFamily                   - Mssql sku family",
./nested/postgres.json:43:                "family": "[parameters('moodleCommon').skuFamily]",
./nested/postgres.json:58:        "documentation6": " skuFamily                   - Postgresql sku family",
./nested/mysql.json:43:                "family": "[parameters('moodleCommon').skuFamily]",
./nested/mysql.json:58:        "documentation6": " skuFamily                   - Mysql sku family",
```

The only use that looks potentially relevant is [./nested/mysql.json:43](https://github.com/Azure/Moodle/blob/c75b6972ec05798a3c3952ddda953af18be465cf/nested/mysql.json#L43) but given that this is not a parameter in the REST API I believe it is not required - https://docs.microsoft.com/en-us/rest/api/mysql/servers/createorupdate#serverpropertiesfordefaultcreate

**Comment by hosungsmsft** [comment]
Yes, I observed this as well, and was going to remove this... I was worried if that might cause a regression, but I highly doubt so.

In fact, the overall DB configuration story in our templates is not very great. It's especially true after we introduced MS SQL as well. The MySQL/PostgreSQL SKUs also changed, so that needs to be incorporated. We might need some overall reorg design on DB configs...
